packages/shared_preferences/lib/shared_preferences_tizen.dart

```diff
@@ -82,9 +83,36 @@ class SharedPreferencesPlugin extends SharedPreferencesStorePlatform {
     return tizen.preference_remove_all() == 0;
   }
```

Comment:
Other implementation uses `clearWithParameters` in `clear` but this implement doesn't. Wouldn't this be a big problem in future?

Reply:
You are right. it should modified before merged.

Reply:
I modified it.